openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -831,9 +857,11 @@ def _execute(
             outputs = self.run(scratch_basepath=ctx.scratch,
                                shared_basepath=ctx.shared)
 
+        analysis_outputs = self.analyse(ctx.shared)
```

Comment:
Just checking - this needs to be here because the file handles need to be closed right?

What happens if it's a dry run and we delete out the files?

Reply:
Yeah it requires the old file handle is closed/flushed etc.

Dry run will probably break this right now...

Reply:
Am I correct in remembering that verbose only ever happens if you call `run` manually? In which case that might not be as much of an issue.

Not introduced in this PR, but shouldn't the `run` call on line 857 pass `kwargs` through? (or at least the relevant kwargs)